src/webgpu/api/operation/buffers/map.spec.ts

```diff
@@ -362,3 +362,134 @@ still present in the mapped buffer.`
     const actual = new Uint8Array(buffer.getMappedRange(...range));
     t.expectOK(checkElementsEqual(actual, new Uint8Array(expected.buffer)));
   });
+
+g.test('mappedAtCreation,mapState')
+  .desc('Test that exposed map state of buffer created with mappedAtCreation has expected values.')
+  .params(u =>
+    u
+      .combine('validationError', [false, true])
+      .combine('afterUnmap', [false, true])
+      .combine('afterDestroy', [false, true])
+      .beginSubcases()
+      .combineWithParams(kSubcases)
```

Comment:
Hm, I don't think the kSubcases are important to this test case. Feel free to keep if you think they might be useful (or if they're very fast), but I think we could omit them and make the test a little faster.

Reply:
Honestly I just followed the other test cases and have added `kSubcases` without deeply thinking. The suggestion sounds good to me. I don't think it's needed in these test cases. I have removed it, thanks.